README.md

```diff
@@ -1,18 +1,44 @@
-# Game On! Auth Service 
+# Spring Security GameOn Auth.
 
-[![Codacy Badge](https://api.codacy.com/project/badge/Grade/bfd8ece69df0405eb5165f00fefc087e)](https://www.codacy.com/app/gameontext/gameon-auth?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=gameontext/gameon-auth&amp;utm_campaign=Badge_Grade)
+Implementation of GameOn Auth project, using Spring Security.
 
-See the [application architecture description](https://gameontext.gitbooks.io/gameon-gitbook/content/microservices/) in the Game On! Docs for more information on how to use this service.
```

Comment:
This section in the book should perhaps be updated, and we could change this line to say "for more information about how this service fits in the overall application architecture."

Reply:
Sounds like something that will need addressing across every GO service, they all share this boilerplate.. separate issue to the spring move.